packages/cli/src/commands/setup/custom-entry/custom-entry.js

```diff
@@ -0,0 +1,67 @@
+import fs from 'fs'
+import path from 'path'
+
+import chalk from 'chalk'
+import Listr from 'listr'
+
+import { getPaths, writeFile } from 'src/lib'
+import c from 'src/lib/colors'
+
+export const command = 'custom-entry'
+
+export const description =
+  'Setup a custom entry.js file, so you can customise how the redwood app is mounted in your browser'
+
+export const builder = (yargs) => {
+  yargs.option('force', {
+    alias: 'f',
+    default: false,
+    description: 'Overwrite existing entry.js file',
+    type: 'boolean',
+  })
+}
+
+export const handler = async ({ force }) => {
+  const tasks = new Listr([
+    {
+      title: 'Creating new entry point in `web/src/entry.js`.',
+      task: () => {
+        // @TODO figure out how we're handling typescript
+        // In this file, we're setting everything to js
+        const entryJsFile = `${getPaths().web.src}/entry.js`
+
+        // Copy over the entry file that exists in package/web
+        // This is the file that's normally used if a custome entry point doesnt exist
+        return writeFile(
+          entryJsFile,
+          fs
+            .readFileSync(
+              path.resolve(__dirname, '../../../../../web/entry/index.js')
+            )
```

Comment:
I usually use `paths.join(getPaths().base, 'node_modules/@redwoodjs/web/entry/index.js)` so that if this file moves around that it's not going to cause problems.

Reply:
hmm sure makes sense